client/components/ViewCatalog.js

```diff
@@ -13,6 +13,7 @@ import { Link } from 'react-router-dom';
 import Snackbar from '@material-ui/core/Snackbar';
 import IconButton from '@material-ui/core/IconButton';
 import CloseIcon from '@material-ui/icons/Close';
+import { SignalWifi1BarLockSharp } from '@material-ui/icons';
```

Comment:
switch import style to match line 15 per material ui import suggestions - only import what you need rather than the entire library